Consoleモジュールのドキュメンテーションを作成 (#218)
MLShukai/pamiq-core: docs/user-guide/console.md

# Console

> The `console` module provides interfaces for user interaction and external system control through both a command-line interface and a web API.

Reviewer: WebAPIの説明を入れたいならどこか別の場所がいいと思う　一番最初では説明対象が何なのかを簡潔に説明することに重点をおくべきで、中で何を使ってどんな仕組みで動いているかが入ってきちゃうと混乱を招くと思う
```suggestion
The `console` module provides the interactive command-line interface. It can control PAMIQ-Core system externally.
```

## PAMIQ Console

After launching a PAMIQ-Core system, you can connect to it using the `pamiq-console` command-line tool:

```sh
$ pamiq-console
Welcome to the PAMIQ console. "help" lists commands.

pamiq-console (active) >
```

> The console prompt shows the current system state (e.g., `active`, `paused`, `shutting down`) in parentheses.

Reviewer: `shutting down`ってstate存在する？`shutdown`実行した時って`shutting down`じゃなく`offline`にならない？
